driver/amc_pci.spec.in

```diff
@@ -15,31 +15,41 @@ Packager: Michael Abbott <michael.abbott@diamond.ac.uk>
 
 %install
 rm -rf %{buildroot}
 mkdir -p %{buildroot}%{dkmsdir} %{buildroot}%{udevdir}
-install -m 0644 %{_sourcedir}/amc_pci_core.c     %{buildroot}%{dkmsdir}
-install -m 0644 %{_sourcedir}/amc_pci_core.h     %{buildroot}%{dkmsdir}
-install -m 0644 %{_sourcedir}/amc_pci_device.h   %{buildroot}%{dkmsdir}
-install -m 0644 %{_sourcedir}/debug.c               %{buildroot}%{dkmsdir}
-install -m 0644 %{_sourcedir}/debug.h               %{buildroot}%{dkmsdir}
-install -m 0644 %{_sourcedir}/dma_control.c         %{buildroot}%{dkmsdir}
-install -m 0644 %{_sourcedir}/dma_control.h         %{buildroot}%{dkmsdir}
-install -m 0644 %{_sourcedir}/error.h               %{buildroot}%{dkmsdir}
-install -m 0644 %{_sourcedir}/interrupts.c          %{buildroot}%{dkmsdir}
-install -m 0644 %{_sourcedir}/interrupts.h          %{buildroot}%{dkmsdir}
-install -m 0644 %{_sourcedir}/memory.c              %{buildroot}%{dkmsdir}
-install -m 0644 %{_sourcedir}/memory.h              %{buildroot}%{dkmsdir}
-install -m 0644 %{_sourcedir}/registers.c           %{buildroot}%{dkmsdir}
-install -m 0644 %{_sourcedir}/registers.h           %{buildroot}%{dkmsdir}
-install -m 0644 %{_sourcedir}/Makefile              %{buildroot}%{dkmsdir}
-install -m 0644 %{_sourcedir}/Kbuild                %{buildroot}%{dkmsdir}
-install -m 0644 %{_curdir}/dkms.conf                %{buildroot}%{dkmsdir}
-install -m 0644 %{_curdir}/11-amc_pci.rules      %{buildroot}%{udevdir}
+install -m 0644 %{_sourcedir}/amc_pci_core.c             %{buildroot}%{dkmsdir}
+install -m 0644 %{_sourcedir}/amc_pci_core.h             %{buildroot}%{dkmsdir}
+install -m 0644 %{_sourcedir}/amc_pci_device.h           %{buildroot}%{dkmsdir}
+install -m 0644 %{_sourcedir}/debug.c                    %{buildroot}%{dkmsdir}
+install -m 0644 %{_sourcedir}/debug.h                    %{buildroot}%{dkmsdir}
+install -m 0644 %{_sourcedir}/dma_control.c              %{buildroot}%{dkmsdir}
+install -m 0644 %{_sourcedir}/dma_control.h              %{buildroot}%{dkmsdir}
+install -m 0644 %{_sourcedir}/error.h                    %{buildroot}%{dkmsdir}
+install -m 0644 %{_sourcedir}/interrupts.c               %{buildroot}%{dkmsdir}
+install -m 0644 %{_sourcedir}/interrupts.h               %{buildroot}%{dkmsdir}
+install -m 0644 %{_sourcedir}/memory.c                   %{buildroot}%{dkmsdir}
+install -m 0644 %{_sourcedir}/memory.h                   %{buildroot}%{dkmsdir}
+install -m 0644 %{_sourcedir}/registers.c                %{buildroot}%{dkmsdir}
+install -m 0644 %{_sourcedir}/registers.h                %{buildroot}%{dkmsdir}
+install -m 0644 %{_sourcedir}/prom_processing.c          %{buildroot}%{dkmsdir}
+install -m 0644 %{_sourcedir}/prom_processing.h          %{buildroot}%{dkmsdir}
+install -m 0644 %{_sourcedir}/default_prom.config        %{buildroot}%{dkmsdir}
+install -d                                               %{buildroot}%{dkmsdir}/tools
+install -m 0755 %{_sourcedir}/tools/prom_data_creator.py %{buildroot}%{dkmsdir}/tools
```

Comment:
I know this line is bigger than 79 but I didn't know how to wrap it in a good way